Doc/library/tkinter.font.rst

```diff
@@ -47,7 +47,16 @@ The different font weights and slants are:
 
    .. method:: actual(option=None, displayof=None)
 
-      Return the attributes of the font.
+      Returns information about the actual attributes that are obtained when
+      the font is used on *displayof*; the values obtained may differ from
+      those given (and retrieved by :meth:`Font.cget`) due to
+      platform-dependent limitations (such as the availability of font
+      families and pointsizes). If *displayof* is omitted, it defaults to the
```

Comment:
Perhaps let's separate the last sentence about `displayof` into a new paragraph, and mention what type of object it is: Is it a string? What does it reference? Can we give an example?